.github/workflows/e2e_tests.yaml

```diff
@@ -26,7 +26,7 @@ on:
 
 jobs:
   android:
-    runs-on: macos-13
+    runs-on: macos-13-xl
```

Comment:
What's the reason to upgrade to large runners? Seems in all other places we are using standard runner.